config-windows.md

```diff
@@ -94,3 +94,23 @@ The following parameters can be specified:
         }
    }
 ```
+
+## <a name="configWindowsHyperV" />HyperV
+
+`hyperv` is an OPTIONAL field of the Windows configuration. If present, the container will be run with Hyper-V isolation. If omitted, the container will be run as a Windows Server container.
+
+The following parameters can be specified:
+
+* **`utilityvmpath`** *(string, OPTIONAL)* - specifies the path to the image used for the utility VM. If not supplied, the runtime will search the container filesystem layers from the bottom-most layer upwards, until it locates "UtilityVM", and default to that path.
+
+* **`sandboxpath`** *(string)* - specifies the root of the path to the sandbox to be used for the container.
```

Comment:
Is this REQUIRED or OPTIONAL?  My very uninformed guess is that it is REQUIRED (if you set `hyperv`, you MUST set `sandboxpath`).  In case it helps (if you like ABNF), I took a stab at explaining the property-definition syntax [here][1] (from #747).

[1]: https://github.com/wking/opencontainer-runtime-spec/blob/05da00bc76e1746276e977ab45957ab39c31c73a/spec.md#property-specification